docs/working-with-deephaven-tables/merging-tables.md

```diff
@@ -0,0 +1,103 @@
+---
+id: merging-tables
```

Comment:
My comments line up to your outline.

I. Intro
2. Syntax 
- include both methods
- example
3. efficient merges